clients/admin-ui/src/features/configure-consent/VendorCookieTable.tsx

```diff
@@ -46,9 +45,9 @@ const VendorCookieTable = () => {
     [systems]
   );
 
-  const [systemToEdit, setSystemToEdit] = useState<System | undefined>(
-    undefined
-  );
+  // const [systemToEdit, setSystemToEdit] = useState<System | undefined>(
```

Comment:
This was to get the build working. This component is effectively deprecated but I'm keeping in the code base just in case